src/app/public/src/modules/footer/footer.component.spec.ts

```diff
@@ -129,7 +129,8 @@ describe('StacheFooterComponent', () => {
     let mappedFooterLinks = footerConfig.nav.items.map((navItem: any) => {
       return {
         name: navItem.title,
-        path: navItem.route
+        path: navItem.route,
+        isCurrent: false
```

Comment:
Why do we need an isCurrent value for footer links?

Reply:
I've been looking in to this and for some reason I cannot immediately identify this the component component is generating a value for this specific test that has the isCurrent and isActive values attached to the generated component.footerLinks. Neither of us has touched this test since I started working on the Stache updates, other than to add those properties to get it running. 

I am going to keep looking in to why this is happening. It feels like a bug.